packages/paywall/src/urls.ts

```diff
@@ -1,4 +1,4 @@
-let baseUrl = 'https://paywall.unlock-protocol.com' // assume prod
+let baseUrl = window?.location?.href || 'https://paywall.unlock-protocol.com' // assume prod
```

Comment:
Please explain why it would break?

The env var is not enough. The goal is to identify automatically without having to use en env var.

Reply:
baseURL is used as fallback in case we don't find the paywall script in the DOM with correct url pointing to staging or production. We use this url to figure out which checkout to display which is always hosted on our url. 

staging-paywall -> staging-checkout
production-paywall -> production checkout
anything else -> localhost currently in case of baseURL being set to non-unlock domain.

Reply:
ok makes sense. I'll clean it up.